dev_tools/configure_vscode_for_bazel.py

```diff
@@ -31,6 +35,35 @@ def parse_arguments(argv: Sequence[str] | None = None) -> argparse.Namespace:
         action="store_true",
         help="Show verbose output.",
     )
+    parser.add_argument(
+        "--build",
+        action="store_true",
+        help="Run recommended bazel build/run actions.",
+    )
+    parser.add_argument(
+        "--generate-launch-json",
+        action="store_true",
+        help="Generate the `launch.json` file.",
+    )
+    parser.add_argument(
+        "--no-generate-launch-json",
+        dest="generate_launch_json",
+        action="store_false",
+        help="Do not generate the `launch.json` file.",
+    )
+    parser.set_defaults(generate_launch_json=True)
+    parser.add_argument(
+        "--generate-compile-commands",
+        action="store_true",
+        help="Generate the `compile_commands.json` file.",
+    )
+    parser.add_argument(
+        "--no-generate-compile-commands",
+        dest="generate_compile_commands",
+        action="store_false",
+        help="Do not generate the `compile_commands.json` file.",
+    )
+    parser.set_defaults(generate_compile_commands=True)
```

Comment:
With the growing argument list and the do/don't opinions, maybe we want to switch to click as argument parser. That should give us a bit more features to implement a nice CLI.

Reply:
I somewhat dislike introducing a new dependency.

There is also https://docs.python.org/3/library/argparse.html#argparse.BooleanOptionalAction, but that is only available from Python 3.9 only.